front-end/src/components/Signup/index.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import styled from 'styled-components';
+import FormInput from '../../shared/FormInput';
+import { withRouter, Link } from "react-router-dom";
+import Logo from '../../svg/Logo';
+import axios from 'axios';
+
+
+
+
+
+class Signup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      /*phoneNumber: '',
+      name: '',
+      username: '',*/
+      email: '',
+      password: '',
+
+    };
+  }
+
+  handleFormSubmit =async (event) =>{
+    event.preventDefault();
+    let payload={
+      "email":this.state.email,
+      "password":this.state.password
+    }
+
+    try{
+      const config = {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }
+
+      const res = await axios.post('http://localhost:3000/users', payload, config);
+      if(res.status === 200){
+        localStorage.removeItem('access_token');
+        localStorage.setItem('access_token', res.data.token);
+        this.props.history.push('/dashboard/');
+      }
+
+      console.log(res);
+
+
+    } catch (err) {
+        console.error(err);
+    }
+
+  }
+
+  handleChange = (field, event) => {
+    this.setState({
+      [field]: event.target.value
+    })
+  }
+
+  render() {
+    return (
+      <SignupContainer>
+        <LeftPanel>
+          <Logo fill="#FFFFFF" height="25%" />
+          <div>
+            Welcome to the Github Connector App
+          </div>
+        </LeftPanel>
+        <RightPanel>
+          <FormContainer>
+            <FormHeading>
+              Create your account
+            </FormHeading>
+            <form onSubmit={this.handleFormSubmit}>
+            {/*<FormInput heading="Username" type="text" value={this.state.username} onChange={(event) => this.handleChange("username", event)} />
+            <FormInput heading="Name" type="text" value={this.state.name} onChange={(event) => this.handleChange("name", event)} />
+            <FormInput heading="Phone Number" type="tel" value={this.state.phoneNumber} onChange={(event) => this.handleChange("phoneNumber", event)} />*/}
```

Comment:
Let's remove any commented code if we don't need it anymore

Reply:
Commented code will be removed